src/main/java/org/jboss/logmanager/formatters/Formatters.java

```diff
@@ -477,6 +483,29 @@ public void renderRaw(Formatter formatter, final StringBuilder builder, final Ex
         };
     }
 
+    private static DateTimeFormatter createDateTimeFormatter(String formatString) {
+        final DateTimeFormatter cached = CACHED_DATE_TIME_FORMATTERS.get(formatString);
+        if (cached != null) {
+            return cached;
+        }
+        if ((formatString == null) || "ISO8601".equals(formatString)) { // Logback uses 'ISO8601'
+            return DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss,SSS");
+        } else if (CONSTANT_PATTERN.matcher(formatString).matches()) {
+            DateTimeFormatter result;
+            try {
+                final MethodHandle methodHandle = MethodHandles.publicLookup()
+                        .findStaticGetter(DateTimeFormatter.class, "ISO_LOCAL_DATE_TIME", DateTimeFormatter.class);
```

Comment:
I assume you meant `formatString` instead of `"ISO_LOCAL_DATE_TIME"`?

Reply:
Yes, just fixed that :) I went too fast and didn't write a test. That is now fixed.